source/includes/markdown/_07-auth.html.md

```diff
@@ -146,8 +146,11 @@ When using the `response_type=code`, your app will receive the following paramet
 | **code** | If response_type=code in the request, this is the code your app can exchange for a token |
 | **state** | The state parameter that was sent with the authorizing request |
 
-You should check that the state is the same in this response as it was in the request. You can read more about the `state`
-parameter [here](https://auth0.com/docs/protocols/oauth2/oauth-state).
+The `state` parameter is necessary to prevent CSRF attacks. As such, **you should check that the `state` is the same in this response as it was in the request.** 
+
+The `state` parameter must contain an unguessable value tied to the user's session, which can be the hash of something tied to their session when the OAuth flow is first initiated (e.g., their session cookie). This value is then passed back and forth between the client application and the OAuth service as a form of CSRF token for the client application. If the `state` parameter is _not_ used, or _not_ tied to the user's session, then attackers can initiate an OAuth flow themselves before tricking a user's browser into completing it. This is similar to a traditional CSRF attack and can also lead to victims binding their accounts to an attacker account without their knowledge. See the [OAuth 2.0 Security best practices](https://tools.ietf.org/id/draft-ietf-oauth-security-topics-13.html) for more detailed information.
+
+For more general information about preventing CSRF attacks with the `state` parameter, feel free to visit [this article](https://auth0.com/docs/secure/attack-protection/state-parameters).
```

Comment:
nit: I'm wondering how this link is different than the link in the previous paragraph. If we want to provide both resources, should we do so in the same paragraph? Or is just one of these sufficient? It feels like we're repeating ourselves a bit

Reply:
I added an Additional Resources section